.changeset/famous-phones-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@coinbase/build-onchain-apps': minor
+---
+
+Add CDP Node as RPC URL
```

Comment:
Make sure to write this as a proper CHANGELOG.

- **feat**: added CDP (Coinbase Developer Portal) Node as RPC URL. You can learn more about this at ~~~~

pretty much you want to give a lot of good information to folks reading the release.

Reply:
added better description

Reply:
```
feat: added CDP (Coinbase Developer Portal) Node as RPC URL. You can learn more about this at https://www.coinbase.com/developer-platform/products/base-node
```